dotCMS/src/main/java/com/dotmarketing/business/ajax/RoleAjax.java

```diff
@@ -937,7 +946,22 @@ public void saveRolePermission(String roleId, String folderHostId, Map<String, S
 
 	}
 
+	/**
+	 * This method send an event to indicate to the angular frame that there is a portlet 
+	 * layouts change, and that the angular frame should refresh the 
+	 * menu options for the users
+	 * 
+	 */
+	private void pushPortletLayoutChangeEvent() {
+
+		SystemEventType systemEventType = SystemEventType.UPDATE_PORTLET_LAYOUTS;
 
+		try {
+	 		systemEventsAPI.push(systemEventType, new Payload(null));
+		} catch (DotDataException e) {
```

Comment:
If there is any reason to catch in upper layers the exception, it could be good. Otherwise as Jose said it might be not necessary. It really depends on the scenario!